src/python/pants/backend/python/rules/pytest_coverage.py

```diff
@@ -329,7 +349,7 @@ async def generate_coverage_report(
     if report_type == CoverageReportType.CONSOLE:
         return CoverageReports((ConsoleCoverageReport(result.stdout.decode()),))
 
-    report_dir = PurePath(coverage_subsystem.report_output_path)
+    report_dir = PurePath(coverage_subsystem.output_dir)
```

Comment:
unrelated to this PR, but I think it would make more sense to have `coverage_subsystem.output_dir` return a `PurePath` instead of a string... if we already wrap the option value...

Reply:
Good idea. Will fix here.